src/pages/workspaces/workspace/analysis/modals/AnalysisModal.ts

```diff
@@ -37,6 +37,8 @@ const titleId = 'analysis-modal-title'
 const analysisMode = Symbol('artifact')
 const environmentMode = Symbol('environment')
 
+const StringSelect = Select as typeof Select<string>
```

Comment:
Ideally, we'd have more specific types, so "StringSelect" and the like wouldn't be a frequent appearance. See CreateGCPBillingProject.ts for an example.